src/Routes.js

```diff
@@ -61,6 +62,14 @@ const renderRoutes = () => (
 );
 
 function Routes() {
+  if(localStorage.getItem('loggedin') == null){ //In place of logged in token will be stored
+    renderRoutes = ()=>(
+      <Switch>
+        <Route path='/login' component={Login}/>
+        <Redirect to='/login'/>
+      </Switch>
+    )
+  }
```

Comment:
This is not a good approach. We should not rely upon local storage.
Since we are using firebase and redux you can fetch the state of the user with `useSelector` hook, and if user ispresent navigate to home else login.

Reply:
Instead of creating render routes here, you can do something like: 
```
if user 
	history.push("/home")
else 
	history.push("/login")
```